openassessment/tests/test_data.py

```diff
@@ -1319,6 +1319,62 @@ def test_create_zip_with_attachments(self):
 
         self.assertTrue(zip_file.read(os.path.join(COURSE_ID, 'downloads.csv')))
 
+    def test_create_zip_with_failed_attachments(self):
```

Comment:
This can probably be decomposed into smaller tests: e.g.
- test a missing file raises an exception
- test a missing file does not appear in the output zip
- test an existing file does appear in the output zip
- test files appear with correct `file_found` tag in downloads.csv

Reply:
I was just following the earlier example on `test_create_zip_with_attachments`. 

I know look closer to E2E but the test when focus entirely on one function which is `create_zip_with_attachments`